meshtastic/mesh_interface.py

```
@@ -799,60 +1504,60 @@ def _get_nested_value(node_dict: dict[str, Any], key_path: str) -> Any:
                 fields = {}
-                for field in showFields:
-                    if "." in field:
-                        raw_value = _get_nested_value(node, field)
+                for col_name in showFields:
+                    if "." in col_name:
+                        raw_value = _get_nested_value(node, col_name)
                     else:
                         # The "since" column is synthesized, it's not retrieved from the device. Get the
                         # lastHeard value here, and then we'll format it properly below.
-                        if field == "since":
+                        if col_name == "since":
```

Ruff (0.15.6)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)